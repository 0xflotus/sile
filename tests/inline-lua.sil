\begin[papersize=a5,class=book]{document}
% A real comment
\begin{script}
answer = tostring(9 % 5)
\end{script}
Modulus: 9 \% 5 = \script{SILE.typesetter:typeset(answer)}

\em{100\% italic}
\end{document}
