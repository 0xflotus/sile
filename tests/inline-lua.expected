Set paper size 	419.5275636	595.275597
Begin page
Mx 	54.8208
My 	76.6203
Set font 	Gentium Plus;10;400;;normal;;LTR
T	48 82 71 88 79 88 86	(Modulus)
Mx 	90.2798
T	29	(:)
Mx 	95.2377
T	28	(9)
Mx 	102.5979
T	8	(%)
Mx 	112.1652
T	24	(5)
Mx 	119.5254
T	32	(=)
Mx 	126.5829
T	23	(4)
Mx 	54.8208
My 	88.6203
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	20 19 19	(100)
Mx 	66.8765
T	8	(%)
Mx 	75.7264
T	76 87 68 79 76 70	(italic)
Mx 	195.4611
My 	519.8019
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20	(1)
End page
Finish
